jablib/src/main/java/org/jabref/logic/preferences/JabRefCliPreferences.java

```diff
@@ -698,9 +687,7 @@ public JabRefCliPreferences() {
         defaults.put(STORE_RELATIVE_TO_BIB, Boolean.TRUE);
 
         defaults.put(AUTOLINK_EXACT_KEY_ONLY, Boolean.FALSE);
-        defaults.put(AUTOLINK_FILES_ENABLED, Boolean.TRUE);
         defaults.put(LOCAL_AUTO_SAVE, Boolean.FALSE);
-        defaults.put(ALLOW_INTEGER_EDITION_BIBTEX, Boolean.FALSE);
```

Comment:
You found an artifact in the code we overlooked when we separated JabRefCli and JabRefGui. The constants about the EntryEditorPreferences called to in the put method here should not reside in JabRefCliPreferences class, but in the JabRefGuiPreferences class. Can you please move them? Thank you.